data/src/main/java/co/nimblehq/data/service/providers/MoshiBuilderProvider.kt

```diff
@@ -0,0 +1,16 @@
+package co.nimblehq.data.service.providers
+
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.adapters.Rfc3339DateJsonAdapter
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import java.util.*
+
+object MoshiBuilderProvider {
+
+    val moshiBuilder: Moshi.Builder
+        get() {
+            return Moshi.Builder()
+                .add(Date::class.java, Rfc3339DateJsonAdapter())
```

Comment:
As I can see, we don't have `Date` in Data class so it's unnecessary. But still thinking about keep it there for usage usage if needed 🤔

Reply:
yup, I keep it here for later usage purpose if needed 👌